mesmerize_napari/cnmf_viz_gui.py

```diff
@@ -52,5 +52,5 @@ def show_eval_gui(self):
 
     def select_contours(self):
         box_size = self.ui.spinBoxBoxSize.value()
-        self.cnmf_viewer.select_contours(box_size=box_size)
+        self.cnmf_viewer.select_contours(box_size=box_size, box_update=True)
```

Comment:
things that relate to updating usually start with the word `update` so I'd rename this kwarg to `update_box`